SplitFiles/game.c

```diff
@@ -131,6 +131,9 @@ void game_do_day(const unsigned int day)
   }
   draw_number(16, 12, day);
   for (i = 0; i < 20000; i++) {;}
+  mustRedraw = 1;
```

Comment:
Not a fan of a global variable arriving from outer space without even a 'G_'-style indicator attached.  But given we're close to shipping I will let it slide!

Reply:
Fair enough, I'll move the extern declaration from the header to the .c file so it's clearer